### PR TITLE
Bump version 1.1.0

### PR DESCRIPTION
**Pull request by RushiT0122**
Bump version 1.1.0

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from setuptools import setup
 
 setup(name='tap-deputy',
-      version='1.0.0',
+      version='1.0.1',
```
Comment (kethan1122) [inline setup.py]: patch version here and minor version bump in changelog.md

**Review by kethan1122 — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (RushiT0122) [reply setup.py]: Resolved the typo.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
 # Changelog
 
+## 1.1.0
+ * Add support for dev mode
```
Comment (kethan1122) [inline CHANGELOG.md]: PR link is missing

**Review by kethan1122 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (RushiT0122) [reply CHANGELOG.md]: Done

**Review by kethan1122 — APPROVED** [review]

**Pull request merged**